fdbserver/workloads/ConfigureDatabase.actor.cpp

```diff
@@ -53,6 +53,9 @@ static const char* backupTypes[] = { "backup_worker_enabled:=0", "backup_worker_
 
 std::string generateRegions() {
 	std::string result;
+#ifdef NO_MULTIREGION_TEST
+	return result;
```

Comment:
If we return here wouldn't the "config" be empty with no information about "usable_regions"?